packages/dds/tree/src/feature-libraries/flex-tree/lazyNode.ts

```diff
@@ -90,10 +90,7 @@ export function makeTree(context: Context, cursor: ITreeSubscriptionCursor): Laz
 		return cached as LazyTreeNode;
 	}
 	const schema = context.schema.nodeSchema.get(cursor.type) ?? fail("missing schema");
-	const output = buildSubclass(context, schema, cursor, anchorNode, anchor);
-	anchorNode.slots.set(flexTreeSlot, output);
```

Comment:
Setting the anchor slot and registering the event also happen in the LazyTreeNode constructor, which is hit by all codepaths of `buildSubclass`, so this is unnecessary.